textattack/commands/attack/attack_args_helpers.py

```diff
@@ -397,10 +382,8 @@ def parse_logger_from_args(args):
     if not args.out_dir:
         current_dir = os.path.dirname(os.path.realpath(__file__))
         outputs_dir = os.path.join(
-            current_dir, os.pardir, os.pardir, os.pardir, "outputs", "attacks"
+            current_dir, os.pardir, os.pardir, os.pardir, "outputs"
         )
-        if not os.path.exists(outputs_dir):
-            os.makedirs(outputs_dir)
         args.out_dir = os.path.normpath(outputs_dir)
```

Comment:
ok, so everything from line 308 to here shouldn't be changed. Something went wrong with your branch on git. It's possible your file is just behind master (could be fixed with a `git pull origin master`) but it might be trickier to revert all these changes.

Message me on Slack if you need help fixing this